sklearn/covariance/outlier_detection.py

```diff
@@ -28,27 +29,38 @@ class OutlierDetectionMixin(object):
         The amount of contamination of the data set, i.e. the proportion
         of outliers in the data set.
 
+    raw_decision : bool
+        Whether or not to consider raw (negative) Mahalanobis distances as the
+        decision function. Otherwise returns `[-(dist)**(1/3) + thres**(1/3)]`.
+        Must be False` (default) for compatibility with the others outlier
+        detection tools.
+
     Notes
     -----
     Outlier detection from covariance estimation may break or not
     perform well in high-dimensional settings. In particular, one will
     always take care to work with ``n_samples > n_features ** 2``.
 
+    The `raw_decision` kwarg now returns the negative Mahalanobis distance as a
+    more appropriate penalty function. The depricated `decision_function` kwarg
+    `raw_values` still returns a postive distance.
+
     """
-    def __init__(self, contamination=0.1):
+    def __init__(self, contamination=0.1, raw_decision=False):
         self.contamination = contamination
+        self.raw_decision = raw_decision
 
-    def decision_function(self, X, raw_values=False):
+    def decision_function(self, X, raw_values=None):
         """Compute the decision function of the given observations.
 
         Parameters
         ----------
         X : array-like, shape (n_samples, n_features)
 
         raw_values : bool
-            Whether or not to consider raw Mahalanobis distances as the
-            decision function. Must be False (default) for compatibility
-            with the others outlier detection tools.
+            (Deprecated) Whether or not to consider raw (positive) Mahalanobis
```

Comment:
We usually just drop the docstring. But sphinx's `.. deprecated` could also be used.